AnnoDesigner.Core/Extensions/StringExtensions.cs

```diff
@@ -54,6 +55,38 @@ public static bool IsPartOf(this string s, IEnumerable<string> tokens)
             return false;
         }
 
+        // This checker is made so it is a 100% sure same Identifiers in name and length of the string.
+        // i discovered that .Contains .startWith and .IsPartOff was not good enough to check this.
+        // (11-06-2022)
+        public static bool IsMatch(this string IdentifierToCheck, List<IBuildingInfo> buildingsToCheck)
```

Comment:
Those 2 new methods are only used inside of `PresetParser`.
Is there a possible future use inside of `AnnoDesigner`?
If not, I would move them to `PresetParser` and adjust some of the naming to comply with the existing conventions.

Reply:
well, i put it there to keep it nice in the programing stuff, so all those commands are in a single file...
But if you want me to make a hole new file for only those 2 then let me know...